Documentation_displayed_on_the_portal/Share_Dataset_Instruction_Page.md

```diff
@@ -0,0 +1,222 @@
+
+- Test that **all** dataset files download correctly from the URLs:
+
+  ```
+  datalad get *
+  ```
+
+#### <a name="doi_obtention"></a> 6) Obtention of a Digital Object Identifier (DOI) for your dataset
+
+Datasets in CONP are required to have a Digital Object Identifier (DOI). A DOI is a unique and permanent identifier associated with a research object to make it citeable and retrievable. To get a DOI for your dataset, follow the following steps:
+
+- Log in to [Zenodo](https://zenodo.org), preferably using your GitHub account.
+
+- Select the new dataset GitHub repository in the [list of GitHub repositories](https://zenodo.org/account/settings/github) of Zenodo.
+
+- Release the new dataset on GitHub (see instructions [here](https://help.github.com/en/articles/creating-releases)). A DOI will automatically be created and it will create an archive of the new dataset on Zenodo. 
+
+- Get the Concept DOI badge from the Zenodo list of GitHub repositories [here](https://zenodo.org/account/settings/github/). Add that DOI to the `README.md` file of your dataset and add its value to the *identifier* field of your `DATS.json` file. This DOI will always link to the DOI of the latest release of the dataset. 
+
+#### <a name="addition_to_conp-dataset"></a> 7) Add the new dataset to the list of CONP datasets
+
+To add the newly created dataset to the list of CONP datasets present in the DataLad conp-dataset super dataset, you will need to submit a pull request to `https://github.com/CONP-PCNO/conp-dataset`. Circle CI will automatically test the newly added dataset to confirm whether files download correctly, validate the format of the `DATS.json` file etc...
+
+**Procedure to follow to add the new dataset to `https://github.com/CONP-PCNO/conp-dataset`:**
+
+- On GitHub, fork a new copy of `https://github.com/CONP-PCNO/conp-dataset` into your userspace.
+
+- Install that copy locally:
+
+	```
+	datalad install https://github.com/<your_user_name>/conp-dataset.git
+	```
+
+- Install the new dataset into the `projects` directory
+
+	```
+	cd conp-dataset/projects
+	git submodule add <new_dataset_github_repository>
+	```
+	*`<new_dataset_github_repository>` will be of the form `https://github.com/<username>/<new_dataset_name>.git`
+	
+- Save the changes and push the branch to your fork
+
+	```
+	datalad save -m '<message>'
+	datalad publish --to origin
+	```
+
+- Send a pull request (PR) from your fork's `master` branch to the `master` branch of `https://github.com/CONP-PCNO/conp-dataset`. You should see 2 file changes in the PR:
+  - Modification to the `.gitmodules` file to add the information for the new dataset. The added information should be of the form:
+
+  	```
+	[submodule "projects/<new_dataset_name>"]
+    	path = projects/<new_dataset_name>
+  		url = https://github.com/<username>/<new_dataset_name>.git
+  	```
```

Comment:
Note that since Joey's format-checking update to require a blank line at the end of files, `.gitmodules` may need manual editing to pass CircleCI; some of the subdatasets I removed and reloaded were added after that terminal blank line.

(Also, do they still need to manually edit the url to put ".git" on the end ?)

Reply:
I checked the URLs with the `git module add` command and it does not need to be manually edited when done this way.

I added a comment about the blank line. Good catch!